lang/LangSource/PyrObject.cpp

```diff
@@ -1374,9 +1373,6 @@ void buildBigMethodMatrix()
 			100. * (double)numFilled/(rowTableSize/sizeof(PyrMethod*)));
 	}
 #endif
-	//post("\t%d method selectors, %d classes\n", numSelectors, numClasses);
-	//post("\tmethod table size %d bytes, ", rowTableSize);
-	//post("big table size %d\n", numSelectors * numClasses * sizeof(PyrMethod*));
 	//postfl("%p %p %p\n", classes, bigTable, sels);
```

Comment:
also these seem relevant, because they will tell you something useful to know about the configuration you are just using.